.hooks/pre-commit.py

```diff
@@ -0,0 +1,22 @@
+import re
+import sys
+
+def main():
+    stage_files = sys.argv[1:]
+    for stage_file in stage_files:
+        conditions = []
+        with open(stage_file, 'r') as f:
+            file_contents = f.readlines()
+            if not file_contents:
+                continue
+            for line in file_contents:
+                if re.match('^[a-zA-Z]', line):
+                    conditions.append(line.split()[0].split(":")[0])
```

Comment:
It is better to use the `strip` methods.
```suggestion
                    conditions.append(line.strip().rstrip(':'))
```

Reply:
done